.travis.yml

```diff
@@ -12,7 +12,7 @@ before_install:
   - sudo apt-get install -yqq texinfo
 
 install:
-  - pip install sphinx==1.3.4
```

Comment:
This was made with intention since sphinx eventually can break the build with newer versions. Let's pin some release.